core/src/autogluon/core/searcher/local_grid_searcher.py

```diff
@@ -13,10 +15,21 @@
 class LocalGridSearcher(LocalSearcher):
     """
     Grid Searcher that exhaustively tries all possible configurations.
-    This Searcher can only be used for discrete search spaces of type :class:`autogluon.space.Categorical`
     """
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
+
+    def __init__(self, grid_numeric_spaces_points_number=4, grid_num_sample_settings: Dict = None, **kwargs):
+        """
+        Parameters
+        ----------
+        grid_numeric_spaces_points_number: int, default = 4
+            number of data point to sample from numeric space
+        grid_num_sample_settings: dict (optional), default = None
+            mapping between numeric space name and number of points to sample from it. Example {'a': 4} means
+            sample 4 points from space 'a'. If no value present in this map, then `grid_numeric_spaces_points_number` will be used.
+        """
+        super(LocalGridSearcher, self).__init__(**kwargs)
```

Comment:
nit: why not just `super().__init__(**kwargs)`

Reply:
fixed